docs/blog/starboard.md

```diff
@@ -0,0 +1,37 @@
+---
+id: blog_starboard
+title: Improving XKS security using Starboard
+---
+
+Just like for all companies today security is top of mind for Xenit.
+As part of improving Xenit Kubernetes Service (XKS), we have the last few months working on implementing [Aqua Securitys](https://www.aquasec.com/) [Starboard](https://github.com/aquasecurity/starboard) to our clusters.
+
+Thanks to this we will be able to quickly generate dashboards with the amount of critical CVE:s in our clusters. This will make it much easier for us to find if our platform container images or our customers for example is
+having issues with the latest [log4j CVE:s](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
+
+We use Starboard to continuously scan all container images for running on on XKS, both from a platform point of view but also our customers images.
```

Comment:
I think you want past tense here. This starts a story. Think "We started using Starboard ..." then the para below says "When we had Starboard up and running, we noticed ..." then the complication "The first issue we found ...", the twist "It turns out ..." and the climax "We helped out ...". Then comes more complications and twists and life-and-death battles with dragons. See also below comment about the end of the story.